modules/ROOT/pages/_partials/endpoint-https-send.adoc

```diff
@@ -57,18 +54,18 @@ Required: No +
 Required: No +
 
 |Basic
-|Simple authentication policy protects the API by requiring username and password when calling apps make a request. +
+|Simple authentication policy, it protects the API by requiring a username and password when calling apps make a request. +
 `Username` - Enter the username you set up with this authentication policy. +
-`Password` - Enter the username you set up with this authentication policy. +
+`Password` - Enter the password you set up with this authentication policy. +
```

Comment:
```suggestion
`Password` is the password you set up with this authentication policy. +
```

Reply:
Convert all the rest to sentences, or use definition list format for such " - " entries:
```suggestion
`Password`
Enter the password you set up with this authentication policy. +
```

Reply:
Apply previous suggestions to all these items. I will not mark them individually, except to copyedit.